bin/node-sass

```diff
@@ -227,6 +230,14 @@ function run(options, emitter) {
     }
   }
 
+  if (options.functions) {
+    if ((path.resolve(options.functions) === path.normalize(options.functions).replace(/(.+)([\/|\\])$/, '$1'))) {
```

Comment:
This is indeed ugly, but without path resolution, user would need to `./` (or `.\` in Windows) before the relative paths.